lib/config.ts

```diff
@@ -6,7 +6,6 @@ export class Configuration extends ConfigBase implements IConfiguration { // Use
 	CI_LOGGER = false;
 	DEBUG = false;
 	TYPESCRIPT_COMPILER_OPTIONS = {};
-	USE_PROXY = false;
```

Comment:
There's no way to disable it - set it or clear it. The problem with disabling is that the credentials will be kept when they are not used, which is somehow a security risk. That's why we've decided to skip the disabling - set or clear only.